tests/test_ec2.py

```diff
@@ -74,20 +74,20 @@ def test_get_tag_value(tags, key, expected):
     assert get_tag_value(tags, key) == expected
 
 
-@pytest.mark.parametrize('inst_name, use_inst_id, username, keyfile, port, use_gateway, expected', [
-    ('ssh_server', False, 'ubuntu', 'key.pem', 22, False, 'ssh ubuntu@{} -i key.pem -p 22'),
-    (None, True, 'ubuntu', 'key.pem', 22, False, 'ssh ubuntu@{} -i key.pem -p 22'),
-    ('ssh_server', False, 'ubuntu', 'key.pem', 22, True, 'ssh -tt ubuntu@{} -i key.pem -p 22 ssh ubuntu@{}'),
-    (None, True, 'ubuntu', 'key.pem', 22, True, 'ssh -tt ubuntu@{} -i key.pem -p 22 ssh ubuntu@{}'),
+@pytest.mark.parametrize('inst_name, use_inst_id, username, keyfile, keypath, port, use_gateway, expected', [
+    ('ssh_server', False, 'ubuntu', 'key.pem', "", 22, False, 'ssh ubuntu@{} -i key.pem -p 22'),
+    (None, True, 'ubuntu', 'key.pem', "", 22, False, 'ssh ubuntu@{} -i key.pem -p 22'),
+    ('ssh_server', False, 'ubuntu', 'key.pem', "", 22, True, 'ssh -tt ubuntu@{} -i key.pem -p 22 ssh ubuntu@{}'),
+    (None, True, 'ubuntu', 'key.pem', "", 22, True, 'ssh -tt ubuntu@{} -i key.pem -p 22 ssh ubuntu@{}'),
 ])
-def test_create_ssh_command(mocker, ec2, inst_name, use_inst_id, username, keyfile, port, use_gateway, expected):
+def test_create_ssh_command(mocker, ec2, inst_name, use_inst_id, username, keyfile, keypath, port, use_gateway, expected):
```

Comment:
@mbaxa PEP8 check is failing because this line exceeds 120 characters.

- https://travis-ci.org/achiku/jungle/builds/126767245